com.haxxed.BasiliskII.yml

```diff
@@ -0,0 +1,44 @@
+app-id: com.haxxed.BasiliskII
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: BasiliskII
+finish-args:
+  - --device=dri
+  - --socket=pulseaudio
+  - --socket=wayland
```

Comment:
is it wayland compatible?

Reply:
I assume so? It doesn't use X directly, the app uses GTK2 and SDL2? So it's up to what they do. (doesn't look like GTK2 does wayland)

Reply:
Yeah, it doesn't work on Wayland (maybe SDL2 part does but not the launcher).

Reply:
Should I be removing the wayland perms then?